add(completion)(v7.9): add tab completion description (#788)
cap-js/docs: tools/index.md

| @sap/eslint-plugin | 2.6.3       |
</pre>

### *cds add completion*

> The `cds` command supports starting with version `7.9` shell completion with `tab` key for several shells and operating systems.

Reviewer: ```suggestion
The `cds` command supports shell completion with <kbd>tab</kbd> key for several shells and operating systems.
```

For Linux, macOS and Windows use the following command to activate the shell completion.
```sh
cds add completion
```

> After that, you have to source or restart your shell and can enjoy shell completion support for all cds commands.

Reviewer: ```suggestion
After that, restart your shell (or source the shell configuration) and enjoy shell completion support for all `cds` commands.
```

Currently supported shells:
| Operating System  | Shell |
|-------------------|-------|
| Linux             | bash, zsh |
| macOS             | bash, zsh |
| Windows           | PowerShell, Git Bash |
| WSL               | bash, zsh |

To remove the shell completion, run the following command
```sh
cds completion --remove
```

> and source or restart your shell afterwards.

Reviewer: ```suggestion
and source or restart your shell afterwards.

Shell completion is available starting with version `7.9.0` of `@sap/cds-dk`.
```

| @sap/eslint-plugin | 2.6.3       |
</pre>

### *cds add completion*
The `cds` command supports shell completion with <kbd>tab</kbd> key for several shells and operating systems.

For Linux, macOS and Windows use the following command to activate the shell completion.
```sh
cds add completion
```

After that, restart your shell (or source the shell configuration) and enjoy shell completion support for all `cds` commands.

Currently supported shells:
| Operating System  | Shell |
|-------------------|-------|
| Linux             | bash, zsh |
| macOS             | bash, zsh |
| Windows           | PowerShell, Git Bash |
| WSL               | bash, zsh |

> To remove the shell completion, run the following command
> ```sh
> cds completion --remove
> ```
> and source or restart your shell afterwards.

Reviewer 2: ```suggestion
To remove the shell completion, run the following command:
```sh
cds completion --remove
```
Then source or restart your shell.
```
